docs/getting_started/advanced_setup_guides.rst

```diff
@@ -290,6 +290,9 @@ First, you need to install the master version of our python client:
 
 Then, the easiest way to get the master version of our web app up and running is via docker-compose:
 
+.. note::
+    If you want to run the web app of the master branch **without** docker, we refer you to our :ref:`development-setup`.
+
```

Comment:
Maybe we think about a bit more what's the result of running server app after install rubrix from master:

```bash
pip install -U "git+https://github.com/recognai/rubrix.git[server]"
```